---
layout: default
title: Center
nav_order: 1
description: "Single-cell proteomics center"
permalink: /
---


# Single-cell proteomics center

<div style="font-size:16px; font-weight: 400; letter-spacing: 1.3px;"> 
We develop mass-spec based technologies for quantifying thousands of proteins and their interactions across thousands of single cells. We apply these technologies to understand emerging behaviors in biological systems and to identify mechanisms that underpin health and disease.

[Read more >>](https://science.sciencemag.org/content/367/6477/512)
</div>

&nbsp;

&nbsp;

## Projects

### [Technologies for analyzing protein in single cells](http://slavovlab.net/research.htm#SCoPE-MS)
Human physiology and pathology arise from the coordinated interactions of diverse single cells. However, analyzing single cells has been limited by the low sensitivity and throughput of analytical methods. DNA sequencing has recently made such analysis feasible for nucleic acids, but single-cell protein analysis remains limited. Mass-spectrometry is the most powerful method for protein analysis, but its application to single cells faces three major challenges: Efficiently delivering proteins/peptides to MS detectors, identifying their sequences, and scaling the analysis to many thousands of single cells. These challenges have motivated corresponding solutions, including SCoPE-design multiplexing and clean, automated, and miniaturized sample preparation. Synergistically applied, these solutions enable quantifying thousands of proteins across many single cells and establish a solid foundation for further advances. Building upon this foundation, the SCoPE concept will enable analyzing subcellular organelles and post-translational modifications while increases in multiplexing capabilities will increase the throughput and decrease cost.

[Read more >>](https://arxiv.org/abs/2004.02069)

&nbsp;


### [Mapping the Transcriptome and Proteome of Human Testis in 3D](https://chanzuckerberg.com/science/programs-resources/humancellatlas/seednetworks/mapping-the-transcriptome-and-proteome-of-human-testis-in-3d/)
This network will map the human testis in 3D using paired single cell analysis by mass-spectrometry, clampFISH, and RNA-sequencing.

[Read more >>](https://chanzuckerberg.com/science/programs-resources/humancellatlas/seednetworks/mapping-the-transcriptome-and-proteome-of-human-testis-in-3d/)



&nbsp;


### [Tracking proteome dynamics in single cells](http://slavovlab.net/research.htm#SCoPE-Dyn)
TOne of the most significant recent developments in biomedical research and engineering is the explosion of technologies for quantifying thousands of genes in single cells. These technologies have transformed our ability to identify cell types and cell states in healthy and diseased human tissues. Our laboratory has contributed to these efforts by pioneering powerful methods for quantifying proteins in single cells at unprecedented scale. For the first time, we can examine single cells at systems level. Yet, these glimpses are static: we can only take one snapshot of a cell, showing us protein type and the amount of each protein, per cell. To overcome this limitaion, we develop methods that will enable us to measure the abundance, the birth and and the death rates of proteins across time. By introducing isotopically-labeled amino acids at different time points over the life of a cell, we create a protein travelogue for each cell. Then, the travelogue of each cell is decoded by mass-spectrometry, thus revealing the protein dynamics underpinning the life and functions of each cell. This technology can move biomedical research from the era of still pictures to the era of dynamic movies, the era when we can finally see and understand the dynamics of protein interactions that animate each cell.

[Read more >>](https://chanzuckerberg.com/science/programs-resources/humancellatlas/seednetworks/mapping-the-transcriptome-and-proteome-of-human-testis-in-3d/)




------------

&nbsp;


## About the single-cell proteomics center

The single-cell proteomics center located at [Northeastern University](https://www.northeastern.edu/) in Boston, MA, USA. Questions about the center should be addressed to [Prof. Nikolai Slavov](https://coe.northeastern.edu/people/slavov-nikolai/).
