---
layout: default
title: Research
nav_order: 3
description: "Single-cell proteomics research projects"
permalink: single-cell-proteomics/research
---
{% include social-media-links.html %}

# Research projects
<div style="font-size:16px; font-weight: 400; letter-spacing: 1.3px;">
The center develops single-cell proteomics technologies and uses them to test different conceptual frameworks and discriminate among different classes of models.  
</div>

&nbsp;


 [![]({{site.baseurl}}/single-cell-proteomics/micrographs/Macrophage-eating-bacteria.jpeg){:width="100%" .center-image}]({{site.baseurl}}/single-cell-proteomics/micrographs/Macrophage-eating-bacteria.jpeg)

&nbsp;

### [Developing technologies for analyzing proteins in single cells](https://slavovlab.net/research.htm#SCoPE-MS)
Human physiology and pathology arise from the coordinated interactions of diverse single cells. However, analyzing single cells has been limited by the low sensitivity and throughput of analytical methods. DNA sequencing has recently made such analysis feasible for nucleic acids, but single-cell protein analysis remains limited. Mass-spectrometry is the most powerful method for protein analysis, but its application to single cells faces three major challenges: Efficiently delivering proteins/peptides to MS detectors, identifying their sequences, and scaling the analysis to many thousands of single cells. These challenges have motivated corresponding solutions, including SCoPE-design multiplexing and clean, automated, and miniaturized sample preparation. Synergistically applied, these solutions enable quantifying thousands of proteins across many single cells and establish a solid foundation for further advances. Building upon this foundation, the SCoPE concept will enable analyzing subcellular organelles and post-translational modifications while increases in multiplexing capabilities will increase the throughput and decrease cost.

[Read more >>](https://doi.org/10.1016/j.cbpa.2020.04.018)

&nbsp;


### [Mapping the Transcriptome and Proteome of Human Testis in 3D](https://chanzuckerberg.com/science/programs-resources/humancellatlas/seednetworks/mapping-the-transcriptome-and-proteome-of-human-testis-in-3d/)
This network will map the human testis in 3D using paired single-cell analysis by mass-spectrometry, clampFISH, and RNA-sequencing.

[Read more >>](https://chanzuckerberg.com/science/programs-resources/humancellatlas/seednetworks/mapping-the-transcriptome-and-proteome-of-human-testis-in-3d/)



&nbsp;


### [Tracking proteome dynamics in single cells](https://slavovlab.net/research.htm#SCoPE-Dyn)
One of the most significant recent developments in biomedical research and engineering is the explosion of technologies for quantifying thousands of genes in single cells. These technologies have transformed our ability to identify cell types and cell states in healthy and diseased human tissues. Our laboratory has contributed to these efforts by pioneering powerful methods for quantifying proteins in single cells at unprecedented scale. For the first time, we can examine single cells at systems level. Yet, these glimpses are static: we can only take one snapshot of a cell, showing us protein type and the amount of each protein, per cell. To overcome this limitation, we develop [methods](http://slavovlab.net/research.htm#SCoPE-Dyn) that will enable us to measure the abundance, the birth and and the death rates of proteins across time. By introducing isotopically-labeled amino acids at different time points over the life of a cell, we create a protein travelogue for each cell. Then, the travelogue of each cell is decoded by mass-spectrometry, thus revealing the protein dynamics underpinning the life and functions of each cell. This technology can move biomedical research from the era of still pictures to the era of dynamic movies, the era when we can finally see and understand the dynamics of protein interactions that animate each cell.

[Read more >>](https://alleninstitute.org/what-we-do/frontiers-group/distinguished-investigators/projects/tracking-proteome-dynamics-single-cells)

[![](http://slavovlab.net/index_files/Funders/pa_frontiers_group_circlelogo_rgb.png){:width="60%" .center-image}](https://news.northeastern.edu/2020/10/08/protein-does-a-lot-more-than-build-muscle-why-dont-we-understand-it-better/)

------------

&nbsp;

&nbsp;

### [Single-cell proteomic identification of novel markers of senescence](https://reporter.nih.gov/search/DeZ7K79i2UWhu-DX12QP5g/project-details/10376580)
Cellular senescence is a stable form of cell cycle arrest associated with pro-inflammatory responses. On the one hand, senescent cells are a barrier for tumorigenesis and promote wound healing and embryogenesis. On the other hand, senescent cells accumulate in aged and diseased tissues, where they impair tissue renewal and contribute to inflammation and disease progression. Identification and characterization of senescent cells in human tissues will contribute to our understanding of human diseases. Thus, mapping senescent cells at the 3-dimensional level and single-cell resolution in human tissues is an important biomedical objective. Accurate mapping of senescent cells requires reliable markers to specifically identify senescent cells. Currently the senescence field has limited markers to unambiguously distinguish between senescent cells and cells in other pathological states. In addition, the available markers do not address the heterogeneity of senescent cells in tissues. To overcome these limitations, we propose to employ a novel single-cell proteomic technology to investigate the proteomes of senescent cells at the single-cell level, with the goal to reveal novel markers of senescence which can be used to identify and map senescent cells in human tissues. Our group has recently developed a novel technology termed Single-Cell ProtEomics by Mass Spectrometry ([SCoPE2](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02267-5)). This platform combines automated cell lysis, improved detection sensitivity, and optimized data acquisition and analyses, allowing detection and quantification of thousands of proteins within a single cell. We have applied this technology to study embryonic stem cell differentiation and macrophage polarization, revealing their heterogeneity and alterations of proteomes at the single-cell level. In this application, we will use the lung as a model system, which accumulate senescent cells that contribute to aging and lung diseases. In the UG3 phase, we will isolate senescent cells from the lungs of naturally aged mice. The isolated senescent cells together with control cells will be subjected to SCoPE2 procedure to quantify their proteomes at the single-cell level, allowing us to create unique signatures for potentially diverse senescent cell populations. This will help discover novel markers of senescent cells that are not possible with traditional technologies. In the UH3 phase, we will apply this technology to human lungs, to validate and to identify new markers of senescence. In aggregate, we will establish new ways of identifying senescent cells that should offer new tools to probe senescent cells in human tissues, facilitating tissue mapping of senescent cells of the SenNet initiative. In addition, this study has the potential to reveal new biology of senescence, addressing the heterogeneity and proteome alterations at the single-cell level.



------------


&nbsp;


## About the single-cell proteomics center

The single-cell proteomics center is located in Boston, MA, USA. Questions about the center should be addressed to [Prof. Nikolai Slavov](https://cos.northeastern.edu/people/slavov-nikolai/).
